actions/access_keys/revoke.go

```diff
@@ -43,3 +43,38 @@ func (a *Action) revoke(c *gin.Context) {
 	contract := mappings.MapToAccessKeyContract(accessKey)
 	c.JSON(http.StatusCreated, contract)
 }
+
+// revoke will revoke the intended model by id
+// Revoke access key godoc
+// @Summary		Revoke access key
+// @Description	Revoke access key
+// @Tags		Access-key
+// @Produce		json
+// @Param		id path string true "id of the access key"
+// @Success		200	{object} models.AccessKey "Revoked AccessKey"
+// @Failure		400	"Bad request - Missing required field: id"
+// @Failure 	500	"Internal server error - Error while revoking access key"
+// @Router		/v1/users/current/keys [delete]
+// @Security	x-auth-xpub
```

Comment:
Same as for `Create access key`: use single method adjust description and use `// @DeprecatedRouter`

Reply:
done